microsoft_intune/plugin.spec.yaml

```
@@ -0,0 +1,77 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: microsoft_intune
+title: "Microsoft Intune"
+description: Integrate Microsoft Intune into your workflow
+version: 1.0.0
+vendor: rapid7
+support: community
```

We'll make this one rapid7 supported since it's popular ask

```suggestion
support: rapid7
```